lib/rules/block-no-empty/README.md

```diff
@@ -23,15 +23,87 @@ a { }
 ```
 
 ```css
-@media print { a {} }
+@media print {
+  a {}
+}
 ```
 
 The following patterns are *not* considered violations:
 
 ```css
-a { color: pink; }
+a {
+  color: pink;
+}
 ```
 
 ```css
-@media print { a { color: pink; } }
+a {
```

Comment:
Let's just keep these two comment examples and remove the other new ones:

```css
a {
  /* foo */
}
```

 ```css
@media print {
  a {
    color: pink;
  }
}
```

I think that's enough to communicate that rules with just comments aren't considered empty by default. Also, the built-in rules are geared towards standard CSS syntax and so we tend to avoid using non-standard syntax (e.g. double slash comments) within the rule docs.